xpdacq/tests/test_beamtimeSetup.py

```diff
@@ -94,9 +94,19 @@ def test_start_beamtime(self):
         self.assertEqual(self.experimenters, self.bt.get('bt_experimenters'))
         self.assertEqual(self.wavelength, self.bt.get('bt_wavelength'))
         self.assertEqual(os.getcwd(), self.home_dir)
-        # FIXME
-        # strtScnLst = ['bt_bt.yml','ex_l-user.yml','sa_l-user.yml','sp_ct_0.1.yml','sp_ct_0.5.yml','sp_ct_1.yml','sp_ct_5.yml','sp_ct_10.yml','sp_ct_30.yml']
-        # self.assertEqual(newobjlist,strtScnLst)
+        # test prepoluate ScanPlan
+        self.assertEqual(len(self.bt.scanplans), len(EXPO_LIST))
+        for sp, expect_arg in zip(self.bt.scanplans, EXPO_LIST):
+            self.assertEqual(sp['sp_args'], (expect_arg,))
+        # test if yml files are saved properly
+        for expo in EXPO_LIST:
+            f_path = os.path.join(glbl.scanplan_dir,
+                                  'ct_{}.yml'.format(expo))
+            self.assertTrue(os.path.isfile(f_path))
+        # test if it can be reloaded
+        for current_sp in self.bt.scanplans:
+            reload_sp = ScanPlan.from_yaml(current_sp.to_yaml())
+            self.assertEqual(reload_sp, current_sp)
```

Comment:
Is there a way to check that `reload_sp` is not the exact same object as `current_sp`? This way we can make certain we loaded it up from the file and didn't short circuit anywhere. (I think hash magic might sort this out, maybe)